test/tstDetailsUtils.cpp

```diff
@@ -52,8 +53,11 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(iota, DeviceType, ARBORX_DEVICE_TYPES)
 
 BOOST_AUTO_TEST_CASE_TEMPLATE(prefix_sum, DeviceType, ARBORX_DEVICE_TYPES)
 {
+  namespace KokkosExt = ArborX::Details::KokkosExt;
```

Comment:
```C++
using ArborX::Details::KokkosExt::exclusive_scan;

Reply:
I thought about it, but decided against it. I want to easily keep track of where `KokkosExt` and what of it is being used.